ReSwift/Utils/TypeHelper.swift

```diff
@@ -11,15 +11,24 @@ import Foundation
 /**
  Method is only used internally in ReSwift to cast the generic `StateType` to a specific
  type expected by reducers / store subscribers.
+ 
```

Comment:
Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)